translator/source_code/combine_wzdx.py

```diff
@@ -0,0 +1,157 @@
+import json
+import copy
+from shapely.geometry.polygon import Polygon
+from shapely.geometry import Point
+import geopy
+from geopy.distance import geodesic
+import pyproj
+import getopt
+import sys
+
+
+def main():
+
+    icone_file, cotrip_file, outputfile = parse_arguments(
+        sys.argv[1:], default_output_file_name='combined_wzdx_message.geojson')
+
+    with open(outputfile, 'w+') as f:
+        wzdx_icone = json.loads(open(
+            icone_file, 'r').read())
+        wzdx_cotrip = json.loads(open(
+            cotrip_file, 'r').read())
+
+        polygon = generate_polygon(
+            wzdx_cotrip['features'][0]['geometry']['coordinates'], 100)
```

Comment:
Why was 100 chosen? Can we add it to a variable so its not a "magic number"?

```suggestion
            polygon_width_in_meters = 100
            wzdx_cotrip['features'][0]['geometry']['coordinates'], polygon_width_in_meters)
```